src/west/_bootstrap/main.py

```diff
@@ -207,12 +209,17 @@ def bootstrap(args):
 
     # Clone the west source code and the manifest into west/. Git will create
     # the west/ directory if it does not exist.
+    tempdir = tempfile.gettempdir()
 
-    clone('manifest repository', manifest_url, manifest_rev,
-          os.path.join(directory, MANIFEST))
+    i = 0
```

Comment:
Looks like this can just be a call to tempfile.mkdtemp():

https://docs.python.org/3/library/tempfile.html#tempfile.mkdtemp

Is there a reason to have the number?

Reply:
Yep, not sure about this one either. The tempfile framework in Python ensures that there is no duplicates

Reply:
Look at code. No folder is created here.
Reason was that creating a folder here would make `clone` fail, if the folder already exists.

But I considered your comment, and changed clone instead, so that it  takes additional argument. Think that looks nicer.

Reply:
Very nice, thanks!